servicetemplates/http%3A%2F%2Fopentosca.org%2Fexample%2Fapplications%2Fservicetemplates/MyTinyToDo-MySql_Docker-w1/SELFSERVICE-Metadata/data.xml

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<selfservice:Application
-        xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice"
->
+<selfservice:Application xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:yml="http://docs.oasis-open.org/tosca/ns/simple/yaml/1.3" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org">
```

Comment:
Why additional namespace declarations? Are they set by Winery? I would remove them as they are not used within the document

Reply:
Yes they are set by winery.

I used the Management UI to adjust the text.
Shall I get rid of these?

Applies to the other comments too.

Reply:
done